### DISCUSSION TITLE
No Collabora office on Android app after update to v7.2.1

### DISCUSSION THREAD

**Discussion by ti-guru** [nextcloud/all-in-one#3413] [Questions] [answered]
After updating to to AIO v7.2.1 (NC 27.1.0) it was not possible to open existing documents or create new ones from the file picker in the android app. However, running the following occ command from this [thread](https://github.com/nextcloud/android/issues/11975) fixed the issue : 

`docker exec --user www-data -it nextcloud-aio-nextcloud php occ config:app:set richdocuments mobile_editing --value="yes"`

Is this something that needs to be included in future versions of the AIO config file, @szaimen, or upstream?

**Comment by szaimen** [comment, marked as answer]
This was already fixed with https://github.com/nextcloud/richdocuments/pull/3184 and needs to get backported to stable27 as well as then released
